tests/test_init.py

```diff
@@ -0,0 +1,565 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the :mod:`pennylane.templates.parameters` module.
+"""
+# pylint: disable=protected-access,cell-var-from-loop
+import pytest
+import numpy as np
+from pennylane.init import (cvqnn_layer_uniform,
+                            cvqnn_layers_uniform,
+                            cvqnn_layer_normal,
+                            cvqnn_layers_normal,
+                            interferometer_uniform,
+                            interferometer_normal,
+                            random_layer_uniform,
+                            random_layers_uniform,
+                            random_layer_normal,
+                            random_layers_normal,
+                            strong_ent_layer_uniform,
+                            strong_ent_layers_uniform,
+                            strong_ent_layer_normal,
+                            strong_ent_layers_normal)
+
+
+@pytest.fixture(scope="module",
+                params=[2, 3])
+def n_subsystems(request):
+    """Number of qubits or modes."""
+    return request.param
+
+
+@pytest.fixture(scope="module",
+                params=[2, 3])
+def n_layers(request):
+    """Number of layers."""
+    return request.param
+
+
+@pytest.fixture(scope="module",
+                params=[None, 2, 10])
+def n_rots(request):
+    """Number of rotations in random layer."""
+    return request.param
+
+
+class TestParsCVQNN:
+    """Tests the pennylane.templates.parameters methods for a cv-quantum neural network."""
+
+    def test_cvqnnlayers_uniform_dimensions(self, n_subsystems, n_layers):
+        """Confirm that pennylane.init.cvqnn_layers_uniform()
+         returns an array with the right dimensions."""
+        a = (n_layers, n_subsystems)
+        b = (n_layers, n_subsystems * (n_subsystems - 1) // 2)
+        p = cvqnn_layers_uniform(n_wires=n_subsystems, n_layers=n_layers, seed=0)
+        dims = [p_.shape for p_ in p]
+        assert dims == [b, b, a, a, a, b, b, a, a, a, a]
+
+    def test_cvqnnlayers_uniform_interval(self, seed):
+        """Confirm that no uniform sample in pennylane.init.cvqnn_layers_uniform() lies outside of interval."""
+        low = -2
+        high = 1
+        p = cvqnn_layers_uniform(n_layers=2, n_wires=10, low=low, high=high, seed=seed)
+        p_uni = [p[i] for i in [0, 1, 2, 4, 5, 6, 7, 9]]
+        assert all([(p_ <= high).all() and (p_ >= low).all() for p_ in p_uni])
+
+    def test_cvqnnlayers_uniform_edgecase(self, seed, tol):
+        """Test sampling edge case of pennylane.init.cvqnn_layers_uniform()."""
+        p = cvqnn_layers_uniform(n_layers=2, n_wires=10, low=1, high=1, mean_active=1, std_active=0, seed=seed)
+        p_mean = np.mean(np.array([np.mean(pp) for p_ in p for pp in p_]))
+        assert np.allclose(p_mean, 1, atol=tol)
```

Comment:
```suggestion
        assert np.allclose(p_mean, 1, atol=tol, rtol=0.)
```

Reply:
When specifying tolerances, we try to explicitly set `rtol=0.`

Reply:
Yep, the reason being is that the default value for `rtol` is _not_ zero, it is `1e-05`.